compose/services/docker-compose.redis.yml

```
@@ -8,5 +8,7 @@
         - "6379:6379"
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 8

(indentation)